Test/HUnit/Lang.hs

```diff
@@ -89,4 +91,8 @@ performTestCase action =
        -- is not the case, please email the maintainer.
        E.Handler (\e -> throw (e :: E.AsyncException)),
 
+#if MIN_VERSION_base(4,9,0)
+       E.Handler (\(E.ErrorCallWithLocation msg _) -> return $ Error Nothing msg),
```

Comment:
Keep in mind that we don't have a proper `Location` attached, but rather this looks like this now:

```
pattern ErrorCall :: String -> ErrorCall 	-- Defined in ‘GHC.Exception’

data ErrorCall = ErrorCallWithLocation String String 	-- Defined in ‘GHC.Exception’
```